eyelid-match-ops/src/encrypted/test/matching.rs

```diff
@@ -0,0 +1,119 @@
+//! Encrypted iris matching tests.
+
+use crate::encoded::{PolyCode, PolyQuery};
+use crate::encrypted::{EncryptedPolyCode, EncryptedPolyQuery};
+use crate::iris::conf::IrisConf;
+use crate::plaintext::test::matching::{different, matching};
+use crate::primitives::poly::Poly;
+use crate::primitives::yashe::Yashe;
+use crate::{EncodeConf, FullBits, FullRes, PolyConf, YasheConf};
+use colored::Colorize;
+
+fn convert_negative_coefficients<C: EncodeConf<PlainConf = FullRes>>(
```

Comment:
Is this a copy of the production function with the same name?

(If we call it from the encryption methods, then we don't have to copy it here, or call it in the tests or benchmarks.)

Reply:
done

Reply:
Thanks!